frontend/src/metabase/dashboard/components/Dashboard/DashboardEmptyState/DashboardEmptyState.tsx

```diff
@@ -37,20 +38,24 @@ export function DashboardEmptyState({
             <Button onlyText onClick={addQuestion}>
               {t`Add a saved question`}
             </Button>
-            {t`, or `}
-            <Link
-              variant="brandBold"
-              to={Urls.newQuestion({
-                mode: "notebook",
-                creationType: "custom_question",
-                collectionId: dashboard.collection_id ?? undefined,
-                cardType: "question",
-              })}
-              className={cx(CS.textBold, CS.textBrand)}
-              onClick={closeNavbar}
-            >
-              {t`ask a new one`}
-            </Link>
+            {!isEmbeddingSdk ? (
```

Comment:
Can you provide a screenshot where this is displayed, it's a bit hard to understand why we need this guard.

Reply:
Ahh great. Thanks!